test/meson.build

```diff
@@ -11,6 +12,15 @@ zimwriter_srcs = [  '../src/zimwriterfs/article.cpp',
                     '../src/tools.cpp']
 
 if gtest_dep.found() and not meson.is_cross_build()
+
+    test_tools_exe = executable('commont-tools-test',
```

Comment:
Why is the icu-i18n lib necessary for the tools?

Reply:
removeAccents() uses it:
```c
  static std::unique_ptr<icu::Transliterator> removeAccentsTrans(icu::Transliterator::createInstance(
      "Lower; NFD; [:M:] remove; NFC", UTRANS_FORWARD, status));
  icu::UnicodeString ustring(text.c_str());
  removeAccentsTrans->transliterate(ustring);
```